packages/zoe/src/contracts/coveredCall.js

```diff
@@ -1,84 +1,94 @@
 // @ts-check
 
 // Eventually will be importable from '@agoric/zoe-contract-support'
-import {
-  swap,
-  assertIssuerKeywords,
-  assertProposalShape,
-} from '../contractSupport';
+import { assertProposalShape, swapExact } from '../contractSupport';
 
+// Import types.
 import '../../exported';
```

Comment:
I like putting this right at the top after the @ts-check. Does that seems like a plausible pattern?

Reply:
I thought eslint wanted `import '@something/...';` first

Reply:
If we use something from npm, eslint will require us to put those imports before a local one like ../../exported. So Chris is right that this pattern is not usable in general, although it is usable here because we don't import anything from npm.